packages/wonder-blocks-modal/src/util/drawer-context.ts

```diff
@@ -1,18 +1,46 @@
 import * as React from "react";
 import type {DrawerAlignment} from "./types";
 
+/**
+ * Centralized default values for the drawer system.
+ *
+ * These constants provide the default behavior for all drawer components
+ * and can be imported by consumers who need to reference or override defaults.
+ */
+
+/** Default duration in milliseconds for drawer slide animations and focus timing. */
+export const DEFAULT_DRAWER_TIMING_DURATION_MS = 400;
+
+/** Default setting for whether drawer animations are enabled. */
+export const DEFAULT_DRAWER_ANIMATED = true;
+
+/** Default setting for whether clicking the backdrop dismisses the drawer. */
+export const DEFAULT_DRAWER_BACKDROP_DISMISS_ENABLED = true;
+
+/** Default value for internal exit animation state. */
+export const DEFAULT_DRAWER_IS_EXITING = false;
+
 export interface DrawerContextProps {
     alignment?: DrawerAlignment;
     animated?: boolean;
     isExiting?: boolean;
     timingDuration?: number;
```

Comment:
Since we return `defaultDrawerContextValue` merged with `context`, could these values be all non-optional (without the `?`)? 

That way, consumers of the context don't need to handle the defaults since the hook/context will handle that logic, and we won't need to export the default values!